source/controller/controller.py

```diff
@@ -103,7 +105,13 @@ def finish_filter(self):
 
     def start_filter(self):
```

Comment:
rename `finish_filter` to `start_project`
rename `start_filter` to `finish_wizard`